testsuite/features/support/twopence_init.rb

```diff
@@ -318,12 +318,13 @@ def client_public_ip(host)
   not_implemented.each do |it|
     return 'NOT_IMPLEMENTED' if node == it
   end
-
+  # Select eth0 interface for ubuntu when deploying on AWS
+  ubuntu_interface = $is_cloud_provider ? 'eth0' : 'ens3'
```

Comment:
Nitpick, we don't need to define it in case it will be never be used.
We could do `$is_cloud_provider ? 'eth0' : 'ens3'` inside the `when  /^debian11/, /^ubuntu/` statement.